CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release Notes
 
 ## Platform Version 0.9.24 - UNRELEASED
-* CLI: Migrate all fluvio crates to `comfy-table` from `prettytable-rs` ([#2285](https://github.com/infinyon/fluvio/pulls/2285))
+* CLI: Migrate all fluvio crates to `comfy-table` from `prettytable-rs` ([#2285](https://github.com/infinyon/fluvio/pull/2285))
```

Comment:
Could you point to the issue instead?

Reply:
Sorry, what do you mean?

Reply:
This is a link to the previous PR. We should add a link to the issue #2263 instead